### ISSUE TITLE
[MediaBundle][Bug] No error messages in user area

### ISSUE THREAD

**Issue by cv65kr** [Kunstmaan/KunstmaanBundlesCMS#1833]
When I try upload file, and ajax request return errors. The right message dosen't show in user area. Example in screenshot.

![bug](https://user-images.githubusercontent.com/9404962/36345042-3ff865d6-1424-11e8-875b-94929965759a.png)

**Comment by Devolicious** [comment]
@cv65kr, good catch. Do you know why it could not save? It would help to reproduce the situation.

**Comment by cv65kr** [comment]
@Devolicious kill me, but I forgot :) but looking at the code, reminds me trouble with mimeTypeGuesser. In free time, i will try reproduce this case, and write something more.

**Comment by cv65kr** [comment]
@Devolicious kill me, but I forgot :) but looking at the code, reminds me trouble with mimeTypeGuesser. In free time, i will try reproduce this case, and write something more.


---------------
UP:
Oh, now i remember. I had misconfiguration in my docker container, and mimeTypeGuesser always return null, entity cannot be save because ```content_type``` field can't be null. Never the less, maybe it is one of those error who never should happen, but IMO we need show correct message for each case.
